Mentioned names restored in export

ryanctan opened the pull request:
Fixes #102. @epicfaace's previously commented out code looks like it's not breaking anymore

ryanctan committed:
Mentioned names restored in export

Previously commented out code looks like it's not breaking anymore

epicfaace committed:
Merge branch 'master' into export-mentioned-names

epicfaace reviewed (changes requested):
Try going to Explore and not filtering by anything, then doing an export. Then you get this error 😢 

![image](https://user-images.githubusercontent.com/1689183/61067100-a9883a80-a3bb-11e9-9885-404cae95781b.png)

ryanctan committed:
mentionedNames exports without filter

Fixes #102, ready to review @epicfaace

mentionedNames data was stored in an array. From that array, the desired data gets stored into the entry for export.

ryanctan commented:
@epicfaace, new fix pushed, please review when you get the chance

epicfaace commented on query.js:
```diff
@@ -543,6 +543,9 @@ exports.projectForEntryList = projectForEntryList
 
 function parseExport(res) {
 
+    var mentions = [];
```
@ryanctan how exactly did these latest changes fix the export?

ryanctan replied:
So I actually don't really know why the commented-out code didn't work, I kinda just combined that code with the code for activities and travels and it worked. Sorry I couldn't be more helpful!

epicfaace replied:
Hmm, code is way more complicated, but it works, so let's merge!
